### ISSUE TITLE
Unknown errors during login - potentially CryptoJS

### ISSUE THREAD

**Issue by hugeblank** [Lemmmy/KristWeb#28]
In theory all websites should work to some extent on mobile browsers, but for some reason on entering my master password, instead of connecting me, like it would be expected to do and does on desktop, it sits there idle and doesn’t seem to bother connecting me to the service. This may just be an issue with iOS devices (like the one I currently am using) but it’s an issue nonetheless, and I would really like to see it resolved. Being able to check my krist balance while my miner is chugging along some distance away from me would be pretty nice. :)

**Comment by hugeblank** [comment]
I'm also having an issue with this on my school chromebooks, although I think that may be related to whatever site blocking software they use. However when I use the "First Time Login" option in the dropdown menu in the top right corner, everything works perfectly as expected. So I have the feeling this may also be the way I could login on iOS, but I can't scroll down to the login button with the First Time login option.

I'll see what I can provide as far as logs.

**Comment by Lemmmy** [comment]
Thanks - it would be very helpful to get the JS errors (if any) from both the iOS device and the Chromebook device. As for the scrolling issue - this should be fixed relatively shortly. It seems Apple adds some extra navbars on top of the browser that the page is unaware of, granting them height that isn't actually visible.

**Comment by Lemmmy** [comment]
Are these issues still occuring? If not, I'll close this issue.

**Comment by hugeblank** [comment]
The issue I reported involving iOS was resolved, scrolling seems to be functional now on that iOS window, which is very good :)

The second issue was me having a fundamental misunderstanding of the application. I failed to realize that kristweb was designed from the ground up to provide support for multiple wallets, and so having to type my password first, and then again for my wallet didn't make sense to me. Now I know that the first time it prompts for a password it's logging into the wallet collection, and the second time I logged in (using the first time login option) was me typing in the password for the actual wallet. Apologies for not clarifying that say... 20 some odd days earlier, my bad.

Closing this issue as the first one brought up is fixed, and my own stupidity and lack of foresight being the second issue... well that's going to be a harder bug to squash.
